components/ai_chat/core/common/DEPS

```diff
@@ -0,0 +1,4 @@
+include_rules = [
+  "+content/public/browser",
```

Comment:
common cannot depend on content.

Reply:
removed in 314e3b1b065f49b21209355c424a5fd557bba091